src/main/scala/no/ndla/frontpageapi/controller/FrontPage.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Part of NDLA frontpage_api.
+ * Copyright (C) 2018 NDLA
+ *
+ * See LICENSE
+ */
+
+// format: off
```

Comment:
Noen grunn til at vi skrur av formatting her?

Reply:
Yess. scalafmt formatterer controllerene på en rar måte, så jeg bare skrudde det av for disse filene.
Det er indenteringen som blir litt rar: 
```scala
  "Get data to display on the front page" **
    GET |>> {
    val frontPage = FrontPageData(List(3254, 3501, 7789, 3954), List())
    Ok(frontPage)
  }
```